assets/fonts/cocogoose/style.css

```diff
@@ -0,0 +1,8 @@
+/* #### Generated By: http://www.cufonfonts.com #### */
+
+    @font-face {
+    font-family: 'COCOGOOSE Regular';
+    font-style: normal;
+    font-weight: normal;
+    src: local('COCOGOOSE Regular'), url('cocogoose_trial.woff') format('woff');
```

Comment:
- [ ] For readability, I would suggest you indent these four lines with two spaces, please.